src/osd/modules/debugger/win/consolewininfo.cpp

```diff
@@ -90,11 +265,19 @@ consolewin_info::consolewin_info(debugger_windows_interface &debugger) :
 }
 
 
+//-------------------------------------------------
+//  dtor
```

Comment:
Please don't do this.  This comment is pure noise.  I can see what a destructor is.